src/styles/blocks/calendar.scss

```
@@ -2,7 +2,6 @@
   width: $daysPerRow * $dayShift;
 
   display: flex;
-  justify-content: center;
```

this should fix your whole setup
```suggestion
  justify-self: center;
```

---

Okay thanx noted. Fixing